### ISSUE TITLE
Error when compiling.

### ISSUE THREAD

**Issue by KirgoPlayer** [arduinocelentano/cdimage#6]
Hi! I downloader Qt and executed qmake in Qt Creator,after this when I run make I get a screen full of errors. Can you try to help me in solving the problem or give me a compiled version of the software?

**Comment by KirgoPlayer** [comment]
> I'm also getting errors when I attempt to compile, but I am unfamiliar with QT so it's possible it's due to my inexperience.

I was able to compile it. I even made a setup out of it.Used the LICENCE file from the source code and credited arduinocelentano.I can share it with his approval.

**Comment by JudyIscariot** [comment]
Yeah if you could share that or even just some instructions on how you arrived at that it'd be super helpful, I'm just lost where qt is concerned.

**Comment by KirgoPlayer** [comment]
> Yeah if you could share that or even just some instructions on how you arrived at that it'd be super helpful, I'm just lost where qt is concerned.

Made a post with the installer on Reddit r/cd_collection

**Comment by arduinocelentano** [comment]
Thank you for your feedback.
Since many Windows users experienced problems with building it or did not want to install Qt and all the stuff, I've made a [Windows x64 binary build](https://github.com/arduinocelentano/cdimage/releases). However I have not tested it thoroughly yet because I don't have a working CD recorder at hand right now. 

@KirgoPlayer, thank you for your effort and interest. Yes, GPL license allows you to distribute and even modify the software as long as you distribute it under the same license terms. If you make binary Windows builds, make sure you include all the linked libraries. I used [windeployqt](https://doc.qt.io/Qt-5/windows-deployment.html) for this purpose.

**Comment by KirgoPlayer** [comment]
> Thank you for your feedback.
> Since many Windows users experienced problems with building it or did not want to install Qt and all the stuff, I've made a [Windows x64 binary build](https://github.com/arduinocelentano/cdimage/releases). However I have not tested it thoroughly yet because I don't have a working CD recorder at hand right now. 
> 
> @KirgoPlayer, thank you for your effort and interest. Yes, GPL license allows you to distribute and even modify the software as long as you distribute it under the same license terms. If you make binary Windows builds, make sure you include all the linked libraries. I used [windeployqt](https://doc.qt.io/Qt-5/windows-deployment.html) for this purpose.

Thanks! I will repost everything here. Deleted it because I didn't want to get into trouble :)

EDIT : I guess with your Windows build mine is usseless now :)

**Comment by milnak** [comment]
https://github.com/arduinocelentano/cdimage/issues/20
